README.md

```diff
@@ -60,6 +57,7 @@ agent = AnyAgent.create(
         instructions="Use the tools to find an answer",
         tools=[search_web, visit_webpage]
     )
+    TracingConfig(output_dir="output") # Optional, but recommended for saving and viewing traces
```

Comment:
💅 nitpick: I would prefer to use `traces` as default (`output` sounds too generic)